### ISSUE TITLE
Why the value of the loss function is very high?

### ISSUE THREAD

**Issue by ibrahimLearning** [luyanger1799/Amazing-Semantic-Segmentation#12]
I used UNet model for segmentation. But the value of the loss function make me confusing. Would you suggest me please, why the loss function look like this:

> Epoch 00437: LearningRateScheduler reducing learning rate to 0.0009275436501333685.
> Epoch 437/2500
> 200/200 [==============================] - 54s 270ms/step - loss: 16968.1371 - mean_io_u: 0.9883
How can I use L2 regularization in UNet model?

Thanks

**Comment by edwardcho** [comment]
Hello Sir,

I met same problems.
![image](https://user-images.githubusercontent.com/29569662/78750050-6ab6fd80-79aa-11ea-967f-661a7f7dab9a.png)

the value of loss is big.

Thanks.

**Comment by ayadav10491** [comment]
Same problem
